scripts/qa_crowbarsetup.sh

```diff
@@ -3964,6 +3964,20 @@ function onadmin_cloudupgrade_reboot_and_redeploy_clients()
     # TODO: restart any suspended instance?
 }
 
+function onadmin_prepare_crowbar_upgrade()
+{
+    if iscloudver 4minus ; then
+        complain 11 "This upgrade path is only supported for Cloud 5+"
+    else
+        # using the API, due to missing crowbar cli integration
+        local digest="--digest -u crowbar:crowbar"
```

Comment:
we are using the digest auth below again, so maybe it makes sense to make it a global variable?